src/framework/customModules/banner/hackableDynamic.js

```diff
@@ -86,8 +94,51 @@ var HackableDynamic = new Module.Class({
         return this._subtitle || '';
     },
 
+    get rotation() {
+        return this._rotation;
+    },
+
+    set rotation(value) {
+        this._rotation = value;
+    },
+
+    get decodefunc() {
+        throw new Error('property not readable');
+    },
+
+    set decodefunc(value) {
+        if (!value) {
+            this._decodefunc = null;
+            return;
+        }
+        // eslint-disable-next-line no-new-func
+        this._decodefunc = new Function('letter', value);
+    },
+
+    // Adapted from https://gist.github.com/EvanHahn/2587465, public domain
     _processText(text) {
         const decomposedText = text.normalize('NFKD');
-        return TextFilters[this.textfilter](decomposedText);
+        let output = '';
```

Comment:
I'd like to see this factored out to an utility function, since it's used in a few different places